libs/platform/src/lib/components/menu/menu-item.component.ts

```diff
@@ -40,7 +40,7 @@ export class MenuItemComponent implements OnDestroy, FocusableOption {
         return this.cascadeDirection === 'left';
     }
```

Comment:
We should have our own strategy using hostAttributes vs HostBindings, so we can unify all this

Reply:
Not sure if this is necessary for this PR. We should address this separately.

Reply:
not at all just a comment.. 

> Not sure if this is necessary for this PR. We should address this separately.